packages/web/docs/src/content/gateway/other-features/custom-plugins.mdx

```diff
@@ -826,6 +826,138 @@ Prefer `onRequestParse` when possible, or wrap the hook code in a `try` block.
 | `serverContext` | The final context object that is shared between all hooks and the GraphQL execution. [Learn more about the context](https://the-guild.dev/graphql/yoga-server/docs/features/context#server-context). |
 | `response`      | The outgoing HTTP response as WHATWG `Response` object. [Learn more about the response interface](https://developer.mozilla.org/en-US/docs/Web/API/Response).                                        |
 
+### `instrumentation`
```

Comment:
Maybe add a general overview of when to use instrumentation? (tracing, ...) 
Should we mention that in most cases, this is a very advanced use-case?